docs/src/changelog.md

```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 <!-- The line "## <square-bracket>Unreleased</square-bracket>" is replaced by the release/release.py script with the new release version and release date. -->
 
+## [Unreleased]
+
+### Added
+
+- Add creationdate to issue trackers. Closes [#2628](https://github.com/ICTU/quality-time/issues/2628).
+
```

Comment:
This should be merged with the release candidate section below, and the release candidate header can be removed.